lib/wick-get-iface-ip

```diff
@@ -33,7 +33,17 @@ wickGetIfaceIp() {
         allInterfaces=true
     fi
 
-    if wickCommandExists ifconfig; then
+    # Works for systemd distros
+    if wickCommandExists networkctl; then
+        cmd=(networkctl status)
+
+	if [[ -n "$interface" ]]; then
+	    cmd[${#cmd[@]}]="$interface"
+	fi
+
+       # FIXME: Unfortunately, IPv4 only
```

Comment:
I can.  I didn't think it was an issue since the current code doesn't support IPv6 or return more than one address (at least, according to my testing) in the list.  I'll include this fix as well.